### PR TITLE
Dev vmwarecloudsimple microsoft.v mware cloud simple 2019 04 01

### PR DESCRIPTION
**Pull request by tenmozes**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by ravbhatnagar** [comment]
This PR has breaking changes. properties have been renamed in the same api-version. confirming with Service owner if this change is needed.

File: specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json [hunk 1]
```diff
@@ -2247,19 +2238,28 @@
...
         },
-        "placementGroupName": {
-          "description": "Placement Name, e.g. \"Placement Group 1\"",
+        "dedicatedPlacementGroupName": {
+          "description": "CloudSimple Placement Name, e.g. \"Placement Group 1\"",
           "type": "string",
           "readOnly": true
         },
+        "nodesCount": {
+          "description": "count of nodes to create",
```
Comment (yungezz) [inline specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json]: count -> Count for consistence.

**Comment by yungezz** [comment]
@ravbhatnagar would you pls add "ArmChangesRequested" label?

**Comment by tenmozes** [comment]
fields will be renamed in next api version

**Pull request closed without merging**